include/api/CFieldDataCategorizer.h

```diff
@@ -78,7 +76,7 @@ class API_EXPORT CFieldDataCategorizer : public CDataProcessor {
     //! Construct without persistence capability
     CFieldDataCategorizer(const std::string& jobId,
                           const CFieldConfig& config,
-                          const model::CLimits& limits,
+                          model::CLimits& limits,
```

Comment:
I thought there was something in `STYLEGUIDE.md` about parameter ordering based on type but I can't find it.. Maybe it was in the Prelert guide? I'll check later.

Reply:
Yes:

> Function parameters MUST be ordered as [in] [in,out] [out]

Nobody else takes any notice of that any more so there are masses of violations of it which is why I didn't bother here, but since you noticed I'll change this one.

Reply:
Actually, come to think of it this one is flukily already in the mandated order, because `limits` is effectively now an `[in,out]` variable.